### PR TITLE
Add grub timeout to autoboot for arch != x86

### PR DESCRIPTION
**Pull request by mattface**
For automating luvos testing, it's handy to not have grub
need a key press to start luvos. Instead, have a 5 second
timeout, unless the arch is x86 where we start bits
immediately.

Also, move the poweroff logic around and remove the sleep
so that the poweroff call happens as soon as the machine
is ready to shutdown.

### CODE REVIEW

**Comment by gkammela** [comment]
In x86 arch, the timeout is set to 0 seconds and thus it doesn't require a key press. Defining a timeout makes difference even if its 0 seconds than not having it at all. I think it will work for ARM too. Please give a try ..

File: meta-luv/classes/luv-efi.bbclass [hunk 1]
```diff
@@ -172,7 +172,9 @@ python build_efi_cfg() {
        cfgfile.write('default=bits\n')
        cfgfile.write('timeout=0\n')
        cfgfile.write('fallback=0\n')
-
+    else:
+       cfgfile.write('default=luv\n')
+       cfgfile.write('timeout=5\n')
```
Comment (ricardon) [inline meta-luv/classes/luv-efi.bbclass]: Wouldn't it be better to simply take the line "cfgfile.write('timeout=0\n') out of the if re.search("(x86_64|i.86)", target): condition?

**Comment by ricardon** [comment]
Hi, @mattface are you planning on making the suggested updates or have any objection on this regard?

**Comment by ricardon** [comment]
Hi @mattface would it be OK for you if I implement the change that I described above?

**Pull request closed without merging**